DataConnectors/Juniper SRX/Connector_Syslog_JuniperSRX.json

```diff
@@ -0,0 +1,123 @@
+{ 
+    "id": "JuniperSRX",
+    "title": "Juniper SRX", 
+    "publisher": "Juniper", 
+    "descriptionMarkdown": "The [Juniper SRX](https://www.juniper.net/us/en/products-services/security/srx-series/) connector allows you to easily connect your Juniper SRX logs with Azure Sentinel This gives you more insight into your organization's network and improves your security operation capabilities.", 
+    "additionalRequirementBanner":"These queries and workbooks are dependent on a parser based on a Kusto Function to work as expected. Follow the steps to use this Kusto functions alias **JuniperSRX** in queries and workbooks. [Follow these steps to get this Kusto functions.](https://aka.ms/sentinelgithubparsersjunipersrx)",
+    "graphQueries": [ 
```

Comment:
@chicduong, could you update the link to  https://aka.ms/sentinel-junipersrx-parser

For future, all the shortlink should be

https://aka.ms/sentinel-[connectorid]-parser  for parser, and
https://aka.ms/sentinel-[connectorid]-azuredeploy for .zip raw azure function.

@v-jayakal, we need to provide the correct link based on the naming schema. Please update the documentation for these as well.

Reply:
updated and noted for future connectors